exotic/exotic.py

```diff
@@ -2984,55 +2991,57 @@ def main():
+            log.info("NOTE: If your file is not in one of the following formats, "
+                     "please re-reduce your data into one of the time formats recognized by EXOTIC.")
+
+            while True:
+                time_format = user_input("Which of the following time formats is your data file stored in? "
+                                         "\nBJD_TDB / JD_UTC / MJD_UTC: ", type_=str)
+                time_format = time_format.upper().strip()
+
+                if time_format not in ['BJD_TDB', 'JD_UTC', 'MJD_UTC']:
                     log.info("Invalid entry; please try again.")
                 else:
-                    formatEntered = True
-            timeFormat = (timeFormat.upper()).strip()
-            goodTimes = timeConvert(goodTimes, timeFormat, pDict, exotic_infoDict)
-
-            #Ask user for flux units and convert to flux if in magnitude/millimagnitude
-            validFluxFormats = ['flux', "magnitude", "millimagnitude"]
-            formatEntered = False
-            log.info("NOTE: If your file is not in one of the following formats, please re-reduce your data into one of the time formats recognized by EXOTIC.")
-            while not formatEntered:
-                fluxFormat = user_input("Which of the following units of flux is your data file stored in? "
-                                        "(Type q to quit)\nflux / magnitude / millimagnitude: ", type_=str)
-                if (fluxFormat.upper()).strip() == 'Q':
-                    sys.exit()
-                elif (fluxFormat.lower()).strip() not in validFluxFormats:
+                    break
+
+            if time_format != 'BJD_TDB':
+                goodTimes = timeConvert(goodTimes, time_format, pDict, exotic_infoDict)
+
+            # Ask user for flux units and convert to flux if in magnitude/millimagnitude
+            log.info("NOTE: If your file is not in one of the following formats, "
+                     "please re-reduce your data into one of the time formats recognized by EXOTIC.")
+
+            while True:
+                flux_format = user_input("Which of the following units of flux is your data file stored in? "
+                                         "\nflux / magnitude / millimagnitude: ", type_=str)
+                flux_format = flux_format.lower().strip()
+
+                if flux_format not in ['flux', 'magnitude', 'millimagnitude']:
                     log.info("Invalid entry; please try again.")
                 else:
-                    formatEntered = True
-            fluxFormat = (fluxFormat.lower()).strip()
-            if fluxFormat != "flux":
-                goodFluxes, goodNormUnc = fluxConvert(goodFluxes, goodNormUnc, fluxFormat)
+                    break
 
-            bjdMidTOld = goodTimes[0]
-            standardDev1 = np.std(goodFluxes)
+            if flux_format != 'flux':
+                goodFluxes, goodNormUnc = fluxConvert(goodFluxes, goodNormUnc, flux_format)
 
```

Comment:
Removed these as it didn't look like we were using them for pre-reduced data. Let me know if this is needed to be placed back in.